api/external/cfgmgmt/request/stats.proto

```diff
@@ -6,6 +6,10 @@ option go_package = "github.com/chef/automate/api/external/cfgmgmt/request";
 message NodesCounts {
   // List of filters to be applied to the node count results.
   repeated string filter = 1;
+  // Earliest node checkin time to return for the node counts.
```

Comment:
i know this is still in draft, i'm just doing some early review..
can you add an example of the format of the time string?

Reply:
also: api requires us to say
- count of nodes that have checked in over the given time period
- total count of nodes that have ever checked in

will the API response, as is, return that information for us?  so in other words, is the total nodes count  returned unaffected by the time range? and then just the success/failure counts are affected by the time range?

Reply:
No, that is not how it is working. The start and end dates are not required. So, to get all the nodes check in ever, send a request with no start and end date. To get the number of nodes that have check in in the last 24 hours send a request with the start with a timestamp from 24 hours ago and the end with the timestamp from now. This would require two requests to get both. 
I will add the example date format to the proto.